src/main/java/uk/gov/companieshouse/ordernotification/config/ApplicationConfig.java

```diff
@@ -71,7 +71,7 @@ Map<DeliveryMethodTuple, String> deliveryMethodMappings(@Value("${email.dispatch
         mappings.put(new DeliveryMethodTuple(DeliveryMethodApi.POSTAL, DeliveryTimescaleApi.STANDARD),
                 "Standard delivery (aim to dispatch within "+dispatchDays+" working days)");
         mappings.put(new DeliveryMethodTuple(DeliveryMethodApi.POSTAL, DeliveryTimescaleApi.SAME_DAY),
-                "Same Day");
+                "Express (Orders received before 11am will be dispatched the same day. Orders received after 11am will be dispatched the next working day)");
```

Comment:
Should there be another full stop before the closing bracket?

Reply:
I will add this in, the ticket does request a full stop